api-spec/protobuf/tdextrade/v2/trade.proto

```diff
@@ -0,0 +1,151 @@
+syntax = "proto3";
+
+package tdextrade.v2;
+
+import "v2/swap.proto";
+import "v2/types.proto";
+import "google/api/annotations.proto";
+
+service Trade {
+  // Markets: List all the markets open for trading.
+  rpc Markets(MarketsRequest) returns (MarketsReply){
+    option (google.api.http) = {
+      get: "/v1/trade/markets"
+    };
+  }
+
+  // Balances: Gets the balances of the two current reserves in the given
+  // market.
+  rpc Balances(BalancesRequest) returns (BalancesReply){
+    option (google.api.http) = {
+      post: "/v1/trade/balance"
+      body: "*"
```

Comment:
should be this maybe a GET call? passing the market as query string?

Reply:
Done